web/src/app/globals.css

```diff
@@ -14,4 +17,29 @@
   }
 }
 
+.Toastify__toast {
```

Comment:
Should move this to a custom component with inline tailwind styles https://fkhadra.github.io/react-toastify/how-to-style/

Reply:
custom component means it has to be inserted on every toast interaction, using CSS is much cleaner way to style missing bits 🙏 

i. e. `toast.info("test")` would become `toast.info(CustomComponent, { data: { content: "test" } })` or `toast.info(<CustomComponent>test</CustomComponent>` which also requires `CustomComponent` import on every use

hence imo `https://fkhadra.github.io/react-toastify/how-to-style/#override-existing-css-classes` is much cleaner